.github/workflows/integration-tests.yml

```diff
@@ -67,14 +99,23 @@ jobs:
 
     steps:
       - name: Check out repository
+        if: github.event_name == 'pull_request'|| github.event_name == 'push'
```

Comment:
I think you need to wrap this in `${{ }}`, like on line 112 in "Update Adapters and Core branches". You can skip it when it's only one check, but once you need some sort of calculation, like the `||`, then you need to wrap it.

Reply:
I'm pretty sure that this is okay, because it was working fine. But I'm certainly no github action expert. Changed.